package.json

```diff
@@ -11,7 +11,7 @@
     "sass": "gulp sass copy-files",
     "test": "NODE_ENV=mocha LOG_LEVEL=OFF mocha -r tsconfig-paths/register $(find src/test \\( -name '*.ts' -or -name '*.js' \\) ! -path '*a11y*' ! -path '*/routes/*') --reporter mochawesome --reporter-options reportFilename=unit,inlineAssets=true,reportTitle=expressjs-template-unit",
     "test:routes": "NODE_ENV=mocha LOG_LEVEL=OFF  TS_NODE_PROJECT=tsconfig.json mocha --require ts-node/register $(find src/test \\( -name '*.ts' -or -name '*.js' \\) -path '*/routes/*') --require ts-node/register --reporter mochawesome --reporter-options reportFilename=routes,inlineAssets=true,reportTitle=expressjs-template-routes",
-    "test:a11y": "NODE_ENV=mocha LOG_LEVEL=ERROR mocha -r tsconfig-paths/register src/test/a11y --timeout 60000 --reporter mochawesome --require ts-node/register --reporter-options reportFilename=a11y,inlineAssets=true,reportTitle=expressjs-template-a11y",
+    "test:a11y": "NODE_ENV=mocha LOG_LEVEL=ERROR TS_NODE_PROJECT=tsconfig.json mocha $(find src/test \\( -name '*.ts' -or -name '*.js' \\) -path '*/a11y/*') --timeout 60000 --reporter mochawesome --require ts-node/register --reporter-options reportFilename=a11y,inlineAssets=true,reportTitle=expressjs-template-a11y",
```

Comment:
Isn't the tsconfig in the default location?

Reply:
we should improve this to have a separate folder for a11y, and just pass in the folder name, this is far more complex than should be needed

Reply:
I tried, but didn't work. wasn't spending time as i am anyway moving to jest and it might just go.